--- work/vmnet-only/filter.c   2013-08-27 20:29:04.000000000 +0100
+++ work/vmnet-only/filter.c   2015-07-07 16:18:32.449649354 -0600
@@ -213,9 +213,7 @@
 #else
                  struct sk_buff **pskb,                // IN:
 #endif
-                 const struct net_device *in,          // IN:
-                 const struct net_device *out,         // IN:
-                 int (*okfn)(struct sk_buff *))        // IN:
+                 const struct nf_hook_state *state)    // IN;
 {
 #ifndef VMW_NFHOOK_USES_SKB
    struct sk_buff *skb = *pskb;
