--- a/js/ui/panel.js	2013-08-08 18:11:11.500815084 -0700
+++ b/js/ui/panel.js	2013-08-08 18:11:32.035942540 -0700
@@ -922,7 +922,6 @@
     'a11y': imports.ui.status.accessibility.ATIndicator,
     'a11yGreeter': imports.ui.status.accessibility.ATGreeterIndicator,
     'volume': imports.ui.status.volume.Indicator,
-    'battery': imports.ui.status.power.Indicator,
     'lockScreen': imports.ui.status.lockScreenMenu.Indicator,
     'keyboard': imports.ui.status.keyboard.InputSourceIndicator,
     'powerMenu': imports.gdm.powerMenu.PowerMenuButton,
