diff -r -u vmmon-only-orig/linux/hostif.c vmmon-only/linux/hostif.c
--- vmmon-only-orig/linux/hostif.c	2013-09-20 00:00:18.768643497 -0700
+++ vmmon-only/linux/hostif.c	2013-09-20 00:00:52.187641920 -0700
@@ -2703,22 +2703,12 @@
       targetHostCpu = vm->currentHostCpu[v];
       if (targetHostCpu != INVALID_PCPU) {
          ASSERT(targetHostCpu < MAX_PCPUS);
-
-#if defined(VMMON_USE_CALL_FUNC)
-         /* older kernels IPI broadcast; use async when possible */
-         (void) compat_smp_call_function(LinuxDriverIPIHandler,
-                                         NULL, VMMON_CALL_FUNC_SYNC);
-
-	 mode = IPI_BROADCAST;
-	 break;
-#else
          /* Newer kernels have (async) IPI targetting */
          arch_send_call_function_single_ipi(targetHostCpu);
 	 mode = IPI_UNICAST;
          if (!all) {
 	    break;
          }
-#endif
       }
       ipiTargets = VCPUSet_Remove(ipiTargets, v);
    }
diff -r -u vmmon-only-orig/linux/vmmonInt.h vmmon-only/linux/vmmonInt.h
--- vmmon-only-orig/linux/vmmonInt.h	2013-09-20 00:00:18.768643497 -0700
+++ vmmon-only/linux/vmmonInt.h	2013-09-20 00:01:29.829640143 -0700
@@ -28,11 +28,7 @@
  * Hide all kernel compatibility stuff in these macros and functions.
  */
 
-#ifdef VMW_HAVE_SMP_CALL_3ARG
 #define compat_smp_call_function(fn, info, wait) smp_call_function(fn, info, wait)
-#else
-#define compat_smp_call_function(fn, info, wait) smp_call_function(fn, info, 1, wait)
-#endif
 
 /*
  * Although this is not really related to kernel-compatibility, I put this
