diff -ur work-orig/kernel/nv-linux.h work/kernel/nv-linux.h
--- work-orig/kernel/nv-linux.h	2013-09-24 22:47:28.514330556 -0700
+++ work/kernel/nv-linux.h	2013-09-24 22:48:17.007309231 -0700
@@ -958,7 +958,11 @@
 #endif
 
 #if !defined(NV_VMWARE)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(3, 11, 0)
 #define NV_NUM_PHYSPAGES                num_physpages
+#else
+#define NV_NUM_PHYSPAGES                (get_num_physpages())
+#endif
 #define NV_GET_CURRENT_PROCESS()        current->tgid
 #define NV_IN_ATOMIC()                  in_atomic()
 #define NV_LOCAL_BH_DISABLE()           local_bh_disable()
